src/PostHog/Api/DecideApiResult.cs

```diff
@@ -17,7 +17,8 @@ internal record DecideApiResult(
     Analytics? Analytics = null,
     bool DefaultIdentifiedOnly = true,
     bool ErrorsWhileComputingFlags = false,
-    IReadOnlyDictionary<string, string>? FeatureFlagPayloads = null);
+    IReadOnlyDictionary<string, string>? FeatureFlagPayloads = null,
+    IReadOnlyList<string>? QuotaLimited = null);
```

Comment:
If I understand the unit tests, `QuotaLimited` isn't ever going to be a part of the successful `/decide` response body. Instead, it will be an error response with the status `HttpStatusCode.PaymentRequired` and a response body that looks like this:

```json
{
    "type": "quota_limited",
    "detail": "You have exceeded your feature flag request quota",
    "code": "payment_required"
} 
```

Correct?

Reply:
Ah, thanks for the clarification! I'll fix things back up. Do you have an example `/decide` response JSON with the quota info?

Reply:
I mean it's just like another `/decide` response, only with empty flags and payloads and this new field.  Looks like this
```
{
  "config": {
    "enable_collect_everything": true
  },
  "toolbarParams": {},
  "isAuthenticated": false,
  "supportedCompression": [
    "gzip",
    "gzip-js"
  ],
  "featureFlags": {},
  "featureFlagPayloads": {},
  "errorsComputingFlags": false,
  "quotaLimited": ["feature_flags"] // it's like this in case we have other quota-limited resources, e.g. "recordings"
  "sessionRecording": false
}
```